itez/beneficiary/tasks.py

```diff
@@ -1,18 +1,18 @@
-import time
 import os
-import sys
-import json
-
 from datetime import datetime
+
 from django.conf import settings
 
+from celery import shared_task
+
 from config import celery_app
 
+from .models import Beneficiary, MedicalRecord
 from .resources import BeneficiaryResource
-from celery import shared_task
+from .utils import GenerateMedicalReport
```

Comment:
lets use relatve imorts here `itez.beneficiary.utils import ...`

Reply:
I have done the same here as well